README.md

```diff
@@ -147,17 +147,17 @@ Skip any step where a compatible tool already exists
 > git clone https://github.com/mbedmicro/mbed
 > pip install virtualenv
 > virtualenv venv
-> 
+>
 ```
 
 Develop
 -------
 1. Update dependencies and start virtual environment. This should be done everytime you pull new changes
 
 ```
-> "venv/Scripts/activate"
+> source venv/bin/activate
```

Comment:
It is, I believe it differs between OS. Prior your change was targetting windows or?